src/access/wac.ts

```diff
@@ -235,3 +319,92 @@ export function getGroupAccessAll(
 ): Promise<AgentWacAccess | null> {
   return getActorAccessAll(resource, getGroupAccessAllWac, options);
 }
+
+async function setActorResourceAccess<T extends WithServerResourceInfo>(
```

Comment:
Kinda nitpick since this is an internal API anyway, but since we're not really distinguishing between Resource and fallback ACLs, I'd prefer to keep these function names aligned with the ACP ones, i.e. just `setActorAccess`.

Reply:
6156b10f217e8dd7a323ebb0c72662878756f28c